stories/topics/con-azure-disaster-recovery.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-azure-disaster-recovery_{context}"]
+
+= Disaster Recovery
+
+When you deploy {AAPonAzureNameShort}, you must enable or disable Disaster Recovery in the `Business Continuity` tab of the form.
+There is no default setting for Disaster Recovery.
+
+The Disaster Recovery feature incurs additional Azure infrastructure costs.
+See xref:con-azure-infrastructure-usage_azure-intro[Azure infrastructure usage] for details of the Service Shape of the Storage account.
+
+
```

Comment:
We should add that once the AAP is deployed, customers who'd like to enable DI should contact customer support in order to do so.

Reply:
Shall we add here a link to the KCS which explains Disaster Recovery in detail? 

https://access.redhat.com/articles/7010302